scripts/hyper-test.sh

```diff
@@ -1 +1 @@
-deno test --allow-net --allow-env --no-check --import-map=https://x.nest.land/hyper-test@0.0.4/import_map.json https://x.nest.land/hyper-test@0.0.4/mod.js
\ No newline at end of file
+deno test --reload --allow-net --allow-env --no-check --import-map=https://raw.githubusercontent.com/hyper63/hyper/tillathehun0/test-migrate-to-using-421/packages/test/import_map.json https://raw.githubusercontent.com/hyper63/hyper/tillathehun0/test-migrate-to-using-421/packages/test/mod.js
```

Comment:
should this be changed back to the published import_map.json

Reply:
Yes, you're correct. The `v2.0.0` of `hyper-test` hasn't been published, so I was just pointing directly to the branch that is PR'd. Once that is merged and released, I will update here. I just wanted to get something pushed up and shown working.

Reply:
ok, I approved